R/run-tests.sh

```diff
@@ -38,6 +38,7 @@ FAILED=$((PIPESTATUS[0]||$FAILED))
 NUM_CRAN_WARNING="$(grep -c WARNING$ $CRAN_CHECK_LOG_FILE)"
 NUM_CRAN_ERROR="$(grep -c ERROR$ $CRAN_CHECK_LOG_FILE)"
 NUM_CRAN_NOTES="$(grep -c NOTE$ $CRAN_CHECK_LOG_FILE)"
+HAS_PACKAGE_VERSION_WARN="$(grep -c "Insufficient package version" $CRAN_CHECK_LOG_FILE)"
```

Comment:
changes to this file is from the other PR plus one change to $NUM_CRAN_NOTES - 1